internal/platform/database/billing/repository.go

```
@@ -143,6 +167,276 @@ func NewRepository(db *sql.DB) *SQLRepository {
+JOIN tenants t ON t.id = b.tenant_id AND t.deleted_at IS NULL
+WHERE b.status = 'overdue'
+  AND b.overdue_notice_count < 3
+  AND b.deleted_at IS NULL
+ORDER BY b.due_date ASC, b.id ASC
+`
+
+	rows, err := r.db.QueryContext(ctx, query)
+	if err != nil {
+		return nil, fmt.Errorf("list overdue reminder candidates: %w", err)
+	}
+	defer rows.Close()
```

golangci-lint (2.11.4)

[error] Error return value of `rows.Close` is not checked

(errcheck)